galaxy/Dockerfile

```diff
@@ -125,6 +125,7 @@ RUN ansible-playbook /ansible/postgresql_provision.yml && \
     --extra-vars galaxy_extras_config_condor=True \
     --extra-vars galaxy_extras_config_condor_docker=True \
     --extra-vars galaxy_extras_config_rabbitmq=True \
+    --extra-vars galaxy_extras_config_cvmfs=True \
```

Comment:
In the last build the monolithic image only failed due to a gpg timeout. The monolithic image has been passing AFAIK.